Resources/Prototypes/Floof/Recipes/Lathes/security.yml

```
@@ -13,3 +13,12 @@
   completetime: 5
   materials:
      Steel: 210
+
+- type: latheRecipe
+  id: ClothingShoesBootsMagCom
+  result: ClothingShoesBootsMagCom
+  completetime: 5
+  materials:
+    Steel: 500
+    Silver: 250
+    Gold: 100
```

```suggestion
  completetime: 10
  materials:
    Steel: 1000
    Plastic: 500
```

Same cost as Science/Normal Magboots, feel free to add extra due to the small armor.